pkgs/development/haskell-modules/configuration-hackage2nix/main.yaml

```diff
@@ -1,40 +1,40 @@
 # pkgs/development/haskell-modules/configuration-hackage2nix.yaml
 
-compiler: ghc-9.0.2
+compiler: ghc-9.2.4
```

Comment:
Ah, good catch here.  I should have known something like this would be necessary.

Reply:
I think we can automate it away (https://github.com/NixOS/nixpkgs/issues/202621). Easy to forget.